### PR TITLE
created tabs component + start on landing implementation

### PR DESCRIPTION
**Pull request by dextmorgn**
## Description
created tabs component + start on landing implementation

### CODE REVIEW

File: src/components/Landing/Tabs/Tabs.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+              onClick={(e) => {
+                handleClick(e, tab.props.label);
+              }}
+            >
+              {tab.props.label}
+            </a>
+          </li>
+        ))}
+      </ul>
+      {/* Fonction map à vérifier */}
```
Comment (dextmorgn) [inline src/components/Landing/Tabs/Tabs.js:31]: check map function here

File: src/components/Landing/Tabs/Tabs.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const [activeTab, setActiveTab] = useState(children[0].props.label);
+  const handleClick = (e, newActiveTab) => {
+    e.preventDefault();
+    setActiveTab(newActiveTab);
+  };
+
+  return (
+    <div className={styles['tabs__module']}>
+      <ul className={styles['tabs__list']}>
+        {/* Fonction map à vérifier */}
```
Comment (dextmorgn) [inline src/components/Landing/Tabs/Tabs.js:14]: check map function here

**Comment by AlexHff** [comment]
Why did you close this?

**Comment by dextmorgn** [comment]
It did not pass the tests and I could not find a way to fix the issue with another commit. 
Next PR is the same but the bug is fixed.

**Pull request closed without merging**